vibe_code_apps_22/scripts/train_four_arms.py

```
@@ -0,0 +1,264 @@
+
+    if index["fail_count"] != 0:
+        if args.ship_desktop:
+            _safe_print("skip --ship-desktop: one or more arms failed")
+        return 1
+
+    if args.ship_desktop:
+        ship_cmd = [sys.executable, "-u", str(ROOT / "scripts" / "ship_best_to_desktop.py")]
+        if args.ship_no_launch:
+            ship_cmd.append("--no-launch")
+        _safe_print(f"=== ship desktop === {' '.join(ship_cmd)}")
+        ship_rc = subprocess.run(ship_cmd, cwd=str(ROOT), check=False).returncode
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)